README.rdoc

```diff
@@ -48,6 +48,10 @@ For initial development, the plugin targets all communication at a vCenter insta
     knife[:vsphere_pass] = "your password"
     knife[:vsphere_dc] = "your-datacenter"
 
+The vSphere password can also be stored in a base64 encoded version (to visually obfuscate it) by prepending 'base64:' to your encoded password. For example:
+
+    knife[:vsphere_pass] = 'base64:VGhhbmtzRXpyYSE=\n'
```

Comment:
I see what you mean:  
```
>> Base64.encode64 "hello"
=> "aGVsbG8=\n"
```

Can you change it to use double quotes? I think that's the least confusing way since it'll match the encoding. Thank you.